scripting/get5.sp

```diff
@@ -999,7 +1004,13 @@ public Action Timer_NextMatchMap(Handle timer) {
   int index = GetMapNumber();
   char map[PLATFORM_MAX_PATH];
   g_MapsToPlay.GetString(index, map, sizeof(map));
-  ChangeMap(map);
+
+  if (g_DisplayGotvVeto.BoolValue) {
+    float minDelay = float(GetTvDelay()) + MATCH_END_DELAY_AFTER_TV;
+    ChangeMap(map, minDelay);
+  } else {
+    ChangeMap(map);
+  }
```

Comment:
Won't this also trigger at the end of map 1 (in a bo3, for example?)

I think the ``minDelay`` variable should be changed in the Event_MatchOver above that creates the timer for ``Timer_NextMatchMap``. Otherwise the delay value is being computed additively in two places, which is needlessly confusing, imo.

Reply:
This is still a behavior change when g_DisplayGotvVeto.BoolValue is false. The ChangeMap call in this function never passed extra delay before, and now it will pass delay when index > 1 now.